code/__DEFINES/dcs/datum_signals.dm

```diff
@@ -122,3 +122,7 @@
 /// Sent when bodies transfer between shades/shards and constructs
 /// from base of /datum/component/construct_held_body/proc/transfer_held_body()
 #define COMSIG_SHADE_TO_CONSTRUCT_TRANSFER "shade_to_construct_transfer"
+
+
+// datum/component/label
+#define COMSIG_LABEL_REMOVE "label_remove"
```

Comment:
Please make this comment in auto-doc format (///) and provide a bit more information in the comment as to when it's fired/where it's called from, similar to other signals in the file!